lib/domain.js

```diff
@@ -16,28 +18,28 @@ const internals = {
 exports.analyze = function (domain, options = {}) {
 
     if (typeof domain !== 'string') {
-        throw new Error('Invalid input: domain must be a string');
+        throw new Error(errorsByCode.DOMAIN_BE_STRING);
```

Comment:
This is developer error, not a validation error. Leave that as a string.

Reply:
Ok. I think the same should be applied to the TLDS option validation.